DynamicActiveRecord.php

```diff
@@ -64,12 +65,13 @@ public static function walk(&$array, $method)
     {
         if (is_scalar($array)) {
             $array = self::$method($array);
+
             return;
         }
 
         $replacements = [];
         foreach ($array as $key => &$value) {
-            if (is_scalar($value)) {
+            if (is_scalar($value) || is_null($value)) {
```

Comment:
i prefer `=== null`.

btw, it's good to know that null is not a scalar type in php. and i'm embarrassed to find that i didn't know it.